host/sgx/sgxsign.c

```diff
@@ -640,6 +642,31 @@ static oe_result_t _init_sigstruct(
         mrenclave,
         sizeof(*mrenclave)));
 
+    if (attributes & SGX_FLAGS_KSS)
+    {
+        if (family_id)
+            OE_CHECK(oe_memcpy_s(
+                sigstruct->isvfamilyid,
+                sizeof(sigstruct->isvfamilyid),
+                family_id,
+                sizeof(*family_id)));
+        if (ext_product_id)
+            OE_CHECK(oe_memcpy_s(
+                sigstruct->isvextprodid,
+                sizeof(sigstruct->isvextprodid),
+                ext_product_id,
+                sizeof(*ext_product_id)));
```

Comment:
Do we have to set `sigstruct->attributemask.flags` to SGX_FLAGS_KSS bit here?

Reply:
#define SGX_SIGSTRUCT_ATTRIBUTEMASK_FLAGS 0XfffffffffffffffbULL
Currently we mask all bits except bit 2 for MODE64BIT so I did not set it but it is a good call and I will set it explicitly.